### DISCUSSION TITLE
Change default key -- 'P' -- for picking under the mouse to other key

### DISCUSSION THREAD

**Discussion by AlexanderG77** [pyvista/pyvista#4951] [Q&A]
How to change the default 'P' key for picking a point on the surface of a mesh, using pyvista.Plotter.enable_surface_point_picking(), to any other key(s)?
I am aware about binding a callback function to a key event with pyvista.Plotter.add_key_event(), but I can not figure out what function is called when pressing 'P' after pyvista.Plotter.enable_surface_point_picking().

**Comment by banesullivan** [comment]
That P key comes directly from the VTK interaction style and isn't easy to change. However, you can add additional keys to trigger the pick event by borrowing some of our internal implementations:

https://github.com/pyvista/pyvista/blob/d72ba085f3a92b80927439061598df9d31f533e9/pyvista/plotting/picking.py#L30-L37

https://github.com/pyvista/pyvista/blob/d72ba085f3a92b80927439061598df9d31f533e9/pyvista/plotting/picking.py#L321-L331

https://github.com/pyvista/pyvista/blob/d72ba085f3a92b80927439061598df9d31f533e9/pyvista/plotting/picking.py#L306-L319

Here's one example:

```py
import pyvista as pv
pl = pv.Plotter(notebook=False)
pl.add_mesh(pv.Sphere())
pl.enable_point_picking()
pl.track_mouse_position()  # continous mouse position tracking (can have performance implications)
pl.add_key_event("t", pl.pick_mouse_position)
pl.show()
```
